src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -476,14 +476,16 @@ def generate_single_bead_permutations(
             If this value is ``None``, :obj:`DEFAULT_BEAM_DIAMETER <MachineConstants.DEFAULT_BEAM_DIAMETER>`
             is used. Valid values are from :obj:`MIN_BEAM_DIAMETER <MachineConstants.MIN_BEAM_DIAMETER>`
             to :obj:`MAX_BEAM_DIAMETER <MachineConstants.MAX_BEAM_DIAMETER>`.
-        min_area_energy_density : float, default: None
-            Minimum area energy density (J/m^2) to use for single bead simulations.
-            Parameter combinations with an area energy density below this value are
-            not included. Area energy density is defined as laser power / (layer thickness * scan speed).
-        max_area_energy_density : float, default: None
-            Maximum area energy density (J/m^2) to use for single bead simulations.
-            Parameter combinations with an area energy density above this value are
-            not included. Area energy density is defined as laser power / (layer thickness * scan speed).
+        min_energy_density : float, default: None
+            Minimum energy density (J/m^3) to use for single bead simulations. The default hatch
```

Comment:
For better logical flow, I would reorder these sentences as follows:

Minimum energy density (J/m^3) to use for single bead simulations. Parameter combinations with an energy density below this value are not included. Energy density is defined as laser power / (layer thickness * scan speed * hatch spacing). The default hatch spacing value of :obj:`DEFAULT_HATCH_SPACING <MachineConstants.DEFAULT_HATCH_SPACING>` is used.